### PR TITLE
[MID-53] add support for initial prompt field

### PR DESCRIPTION
**Pull request by ejscunha**

### CODE REVIEW

**Review by emoragaf — APPROVED** [review]

**Review by ZuraGuerra — APPROVED** [review]

**Comment by ZuraGuerra** [comment]
Does it let you merge, @ejscunha ? It seems to require three approvals

**Comment by ejscunha** [comment]
It needs code owners review which are not set.. I will remove that for now

**Pull request merged**
